src/app/for-nonprofits/page.tsx

```diff
@@ -0,0 +1,116 @@
+"use client";
+import "@/decisions-card/DecisionCriteriaCard";
+import { Container, Image, SimpleGrid, Text, Title, Stack } from "@mantine/core";
+import Banner from "@/components/Banner/Banner";
+import "@mantine/core/styles.css";
+import Footer from "@/components/Footer/Footer";
+import { DecisionCard } from "@/decisions-card/DecisionCriteriaCard";
+
+export default function ProjectsPage() {
+  return (
+    <div>
+      <Banner
+        title="For Nonprofits"
+        text="As a partner, Blueprint will work with you to develop techonology that turns your vision into a reality,
+         completely free of charge!"
+      />
+      <Stack style={{padding: "2vw 1vw"}} gap="sm">
+        <Container className="top-section"
+        style={{display: "flex",
+                flexDirection: "column",
+                alignItems: "center",
+                justifyContent: "center",
+                color: "#e7f3ff",
+                textAlign: "center"}}>
+          <Title
+          style={{color: "#0078e8",
+                  fontFamily: "Poppins",
+                  fontWeight: 600,
+                  fontSize: "3vw",
+                  zIndex: 1,
+                  padding: "10% 0 10%",
+                  lineHeight: "normal"}}>
+            Decision Criteria
+          </Title>
+        </Container>
+        <SimpleGrid
+          cols={{base: 1, sm: 2}} spacing="xs"
+          style={{ maxWidth: '80vw', margin: 'auto' }}
+        >
+          <div>
+            <center>
+              <Image
+                style={{
+                  maxWidth: "70%",
+                  height: "auto",
+                  objectFit: "contain",
+                  position: "relative",
+                  alignItems: "end"}}
+                src="/about_page/placeholder.png" />
+            </center>
+          </div>
+          <div>
+            <center>
+              <Text
+                style={{color: "#0078e8",
```

Comment:
This text should get center formatted when we vertically align it with the image